apps/antidote_crdt/src/antidote_crdt_map_go.erl

```diff
@@ -148,8 +148,7 @@ is_operation(Operation) ->
         {update, {{_Key, Type}, Op}} ->
             antidote_crdt:is_type(Type) andalso antidote_crdt:is_operation(Type, Op);
         {update, Ops} when is_list(Ops) ->
-            distinct([Key || {Key, _} <- Ops]) andalso
```

Comment:
Dialyzer is saying that `distinct\1` is now not used any more. I suggest to completely remove it.